src/styles/search-bar.css

```diff
@@ -0,0 +1,69 @@
+.search-bar {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
you can move `box-sizing: border-box;` to * selector to apply it for all elements